SimpleNtuplizer/plugins/SimpleNtuplizer.cc

```diff
@@ -38,7 +38,8 @@ SimpleNtuplizer::SimpleNtuplizer(const edm::ParameterSet& iConfig):
   superClustersEEToken_(consumes<reco::SuperClusterCollection>(iConfig.getParameter<edm::InputTag>("superClustersEE"))),
   ecalRecHitEBToken_(consumes<edm::SortedCollection<EcalRecHit>>(iConfig.getParameter<edm::InputTag>("ecalrechitsEB"))),
   ecalRecHitEEToken_(consumes<edm::SortedCollection<EcalRecHit>>(iConfig.getParameter<edm::InputTag>("ecalrechitsEE"))),
-  pfLabel_(consumes<reco::PFClusterCollection>(iConfig.getParameter<edm::InputTag>("pfLabel")))
+  pfLabel_(consumes<reco::PFClusterCollection>(iConfig.getParameter<edm::InputTag>("pfLabel"))),
+  pspfLabel_(consumes<reco::PFCluster::EEtoPSAssociation>(iConfig.getParameter<edm::InputTag>("pspfLabel")))
```

Comment:
I thought pspfLabel was the same as pfLabel by construction. Why are you using different tags?